renovate.json

```diff
@@ -1,17 +1,44 @@
 {
-  "baseBranches": ["next"],
   "constraints": {
     "python": "3.7"
   },
   "enabledManagers": ["pip_requirements", "poetry"],
   "extends": [
     "config:base"
   ],
+  "labels": [
+    "dependencies"
+  ],
   "packageRules": [
     {
-      "matchPackageNames": ["graphene-django-optimizer", "django-timezone-field", "django-taggit"],
+      "description": "Require dashboard approval for non-patch updates to graphene-django-optimizer and django-timezone-field.",
+      "matchPackageNames": ["graphene-django-optimizer", "django-timezone-field"],
+      "matchUpdateTypes": ["major", "minor"],
+      "dependencyDashboardApproval": true
+    },
+    {
+      "description": "Require dashboard approval for major updates for django-taggit.",
+      "matchPackageNames": ["django-taggit"],
+      "matchUpdateTypes": ["major"],
       "dependencyDashboardApproval": true
+    },
+    {
+      "description": "Group updates to next cut down on PR noise.",
```

Comment:
Is this how we want to handle multiple release branches? `next` will be getting updates from `develop` so it seems like we either want to merge the updates into `next` and `develop` at the same time (so we don't have to go back on friday to figure out what we merged into develop that week) or just wait for merges from `develop` into `next`?

Reply:
If I'm thinking about this correctly:

- Develop will be getting patch release updates in it's lock file only
- Next will get both major/minor release updates in it's TOML file as well as patch release updates in it's lock file

If both Develop and Next are on the same minor release, yes this might incur an unnecessary lock file collision, but next should always win when merging (in truest sense a rebase) develop into next. It contains updates from much more than just those patch updates. Odds are many dependencies in next are a minor or even a major release ahead of develop.